### PR TITLE
`jellyfish-wallet-classic`

### LINKED ISSUE #452: `jellyfish-wallet-classic` single keypair provider
#### What would you like to be added:

A single keypair provider similar to the mock here: 

https://github.com/DeFiCh/jellyfish/blob/main/packages/jellyfish-wallet/__tests__/node.mock.ts

#### Why is this needed:

Useful for light implementations and e2e tests

### PR DESCRIPTION
**Pull request by monstrobishi**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

WalletClassic implements a simple, single elliptic pair wallet.

#### Which issue(s) does this PR fixes?:

Fixes #452

### CODE REVIEW

**Review by ivan-zynesis — COMMENTED** [review]
Overall looks good. But I am hesitant the import of `jellyfish-wallet` for the WalletEllipticPair interface, this package suppose to much lighter.

I think this classic package user are usually backend microservice which directly accessing their corresponding private key, I am thinking if we can just implement `EllipticPair` instead.

**Comment by monstrobishi** [comment]
@ivan-zynesis I agree but this would require some refactoring of Whale, as Whale requires a WalletEllipticPair interface for it's providers

**Comment by fuxingloh** [comment]
> @ivan-zynesis I agree but this would require some refactoring of Whale, as Whale requires a WalletEllipticPair interface for it's providers

Yeah, it requires `jellyfish-wallet` as `jellyfish-wallet-classic` is a jellyfish wallet provider.

File: packages/jellyfish-wallet-classic/src/classic.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  TransactionSegWit
+} from '@defichain/jellyfish-transaction'
+import { SignInputOption, TransactionSigner } from '@defichain/jellyfish-transaction-signature'
+
+/**
+ * WalletClassic extends WalletEllipticPair with a simple classic implementation.
+ *
+ * Single elliptic pair wallet.
+ */
+export class WalletClassic implements WalletEllipticPair {
```
Comment (fuxingloh) [inline packages/jellyfish-wallet-classic/src/classic.ts:15]: Possibly, in the future we can include these static helper method to make it easier to create Classic Instance

1. `WalletClassic.fromWIF(string)`
2. `WalletClassic.fromPrivkey(Buffer)`

**Review by fuxingloh — APPROVED** [review]

**Review by ivan-zynesis — APPROVED** [review]

**Comment by fuxingloh** [comment]
it just tick over to 97.5, we have 98% coverage now on the badge :)

**Pull request merged**